### DISCUSSION TITLE
Hide IP and port of connected servers from website

### DISCUSSION THREAD

**Discussion by garmouchy** [gameserverapp/Platform#746] [Suggestions]
Hello, 
When you are on the homepage all the servers show up at the bottom with each of their IPs and PORTs. It says click to join but it would be nice to have the ability to hide this information from whoever that visits the website. Yes the IP and port is attainable by other means but most ‘kids’ don’t know where to look. Providing the info and just handing it to them so effortlessly like this would make even the non attackers want to try it. It would be very convenient to have a check box to hide that info or uncheck that whole section. If there are costs involved in the process of making that happen I’m down to chip in.

**Comment by gijoe0618** [comment]
I agree I would really like a way to hide this info as well. It's just making it super easy for ddossers

**Comment by mvaessen** [comment]
Thanks for the suggestion!

You can add this code to the `custom css` section on your community website to hide the info:
```
body.v3 .server-block .server-block-content .info {
display:none;
}
```
https://docs.gameserverapp.com/dashboard/community/website#custom-css

Keep in mind that hiding the IP is absolutely no adequate method to avoid DDoS attacks. The `Connect now` button also has the IP listed. And there are various vote sites that list the IP of the server to the world. 
My advise would be to reconsider whether this really makes a difference vs how this will make it harder for players to easily access your game server.

**Reply by gijoe0618** [reply]
yeah i get what you are saying i just feel like they can search the server name instead, idk i just dont like the IP there thats just me. Thank you so much though for the info on how to hide it, imma do that right now
